deliver/lib/deliver/upload_metadata.rb

```diff
@@ -111,6 +111,7 @@ def upload(options)
       end
 
       v.release_on_approval = options[:automatic_release]
+      v.toggle_phased_release(enabled: !!options[:phased_release])
```

Comment:
In current behavior, if `phased_release` value is not specified on deliver, values on iTC is disabled automatically.

Do you mean If value is not specified, deliver should not change value on iTC?